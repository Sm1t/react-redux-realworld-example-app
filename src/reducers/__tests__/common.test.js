import { expect } from '@jest/globals';
import {
  APP_LOAD,
} from '../../constants/actionTypes';

import commonReducer from '../common'

describe('Reducers: common', () => {
  it('should properly handle action without payload', () => {
    const result = commonReducer({}, { type: APP_LOAD })

    expect(result).toMatchSnapshot()
  });
});