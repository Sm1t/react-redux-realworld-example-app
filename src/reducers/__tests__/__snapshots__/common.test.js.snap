// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reducers: common should properly handle action without payload 1`] = `
Object {
  "appLoaded": true,
  "currentUser": null,
  "token": null,
}
`;
